SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON

```diff
@@ -23,6 +23,7 @@ python-iniparse
 SPEC files that are derived from Fedora SPEC files, which are under the MIT license unless specifically noted otherwise (https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#License_of_Fedora_SPEC_Files):
 c-ares
 chkconfig
+conntrack-tools
```

Comment:
I think the purpose of this file is to document specs that we derived from Photon, but aren't Apache 2.0 licensed due to their original provenance. Since conntrack-tools.spec doesn't seem to be from Photon, it should probably not be added to this file.

Reply:
Good catch, this spec isn't in Photon, I've removed this line. Thanks for pointing it out!